meson.build

```diff
@@ -183,21 +183,28 @@ features = []
 # Directories
 #-------------------------------------------------------------------------------
 
+prefixdir = get_option('prefix')
 sysconfdir = get_option('sysconfdir')
-cdata.set_quoted('SYSCONFDIR', sysconfdir)
-
-bindir = get_option('prefix') / get_option('bindir')
-includedir = get_option('prefix') / get_option('includedir')
-libdir = get_option('prefix') / get_option('libdir')
+bindir = prefixdir / get_option('bindir')
+includedir = prefixdir / get_option('includedir')
+libdir = prefixdir / get_option('libdir')
+datadir = prefixdir / get_option('datadir')
```

Comment:
nice... this makes it more consistent and we don't print in the summmary:

```
$ meson setup -Ddatadir=share2 build5
  User defined options
    datadir       : share2
```
which is confusing, although it does the right thing and on installation creates /usr/share2.